src/rnn.jl

```diff
@@ -159,32 +159,37 @@ function cudnnGetFilterNdDescriptor(wDesc::FD; nbDimsRequested = 8)
     end
 end
 
+
+# call gpu everytime to reflect device changes
+gethandle() = gpu() >= 0 ? cudnnhandle() : nothing
```

Comment:
If this is how we want to handle cpu, should I make this the default behavior of cudnnhandle?

Reply:
OK, I made this the default behavior of cudnnhandle in the latest master.